applications/sudokumb/WebLib/DatastoreCounter.cs

```diff
@@ -184,8 +184,8 @@ public async Task CondenseOldCounters(CancellationToken cancellationToken)
                 Projection = { "__key__" }
             };
             var lazyResults = _datastore.RunQueryLazilyAsync(query,
-                callSettings: callSettings).GetEnumerator();
-            while (await lazyResults.MoveNext())
+                callSettings: callSettings).GetAsyncEnumerator(cancellationToken);

```

Comment:
If we move to .NET Core 3.1 we can use await foreach, which would be much nicer.